source/javascripts/components/_AddStep.js.erb

```diff
@@ -366,6 +366,16 @@
 							category.paginateStepsConfig.currentOffsetInPage
 				);
 			};
+
+			viewModel.search = function() {
+				if (viewModel.stepFilter) {
+					stepSourceService.searchOfficialSteps(viewModel.stepFilter).then(function(stepIds) {
+						viewModel.filteredStepIds = stepIds;
+					});
```

Comment:
don't we need to do error handling here? It might be just ok but double checking :)